packages/twenty-front/src/modules/object-record/record-board/utils/getBeforeAndAfterIndexRecordIds.ts

```diff
@@ -0,0 +1,29 @@
+type getBeforeAndAfterIndexRecordIds = {
+  draggedRecordIndex: number;
+  otherRecordIds: string[];
+};
+type getBeforeAndAfterIndexRecordIdsReturnType = {
+  recordBeforeId?: string;
+  recordAfterId?: string;
+};
+export const getBeforeAndAfterIndexRecordIds = ({
```

Comment:
I would improve the naming with more context, because out of context it's difficult to understand.

Reply:
Thanks for the review @lucasbordeau 
In my opinion we could also make this function very low level and record agnostic such as, `getIndexBeforeAndAfterElementsInArray` or `getIndexNeighboursElementsInArray`

Reply:
Sure why not

Reply:
Done